e2e/testcases/marriage/details-page-validation.spec.ts

```diff
@@ -0,0 +1,147 @@
+import { expect, test } from '@playwright/test'
+import { createPIN, login } from '../../helpers'
+
+test.describe('6. Validate Marriage details page', () => {
+  test.beforeEach(async ({ page }) => {
+    await login(page, 'k.mweene', 'test')
+    await createPIN(page)
+    await page.click('#header_new_event')
+    await page.getByText('Marriage', { exact: true }).click()
+    await page.getByText('Continue', { exact: true }).click()
+    await page.getByText('Continue', { exact: true }).click()
+    await page.getByPlaceholder('dd').fill('02')
+    await page.getByPlaceholder('mm').fill('03')
+    await page.getByPlaceholder('yyyy').fill('1995')
+    await page.getByText('Continue', { exact: true }).click()
+    await page.getByPlaceholder('dd').fill('01')
+    await page.getByPlaceholder('mm').fill('05')
+    await page.getByPlaceholder('yyyy').fill('1993')
+    await page.getByText('Continue', { exact: true }).click()
+  })
+
+  test('1.1. Enter date Less than the current date But after Groom and Bride DOB', async ({
+    page
+  }) => {
+    const pastDate = new Date()
+    pastDate.setDate(new Date().getDate() - 2)
+    const [yyyy, mm, dd] = pastDate.toISOString().split('T')[0].split('-')
+
+    await page.getByPlaceholder('dd').fill(dd)
+    await page.getByPlaceholder('mm').fill(mm)
+    await page.getByPlaceholder('yyyy').fill(yyyy)
+    await expect(
+      page.getByText('Illegal age of marriage', { exact: true })
+    ).toBeHidden()
+    await expect(
+      page.getByText('Required for registration. Enter a valid date', {
+        exact: true
+      })
+    ).toBeHidden()
+  })
+
+  test('1.2. Enter future date', async ({ page }) => {
+    const futureDate = new Date()
+    futureDate.setDate(new Date().getDate() + 5)
+    const [yyyy, mm, dd] = futureDate.toISOString().split('T')[0].split('-')
+
+    await page.getByPlaceholder('dd').fill(dd)
+    await page.getByPlaceholder('mm').fill(mm)
+    await page.getByPlaceholder('yyyy').fill(yyyy)
+    await page.locator('#typeOfMarriage').click()
+    await expect(page.locator('#marriageDate_error')).toHaveText(
+      'Required for registration. Enter a valid date'
+    )
+  })
+  test('1.3. Keep field as null', async ({ page }) => {
+    await page.getByText('Continue', { exact: true }).click()
+    await page.getByText('Continue', { exact: true }).click()
+    await page.getByText('Continue', { exact: true }).click()
+    await page.getByText('Continue', { exact: true }).click()
```

Comment:
There is a helper function [`goToSection()`](https://github.com/opencrvs/opencrvs-farajaland/blob/534aebdf6e66a6451ccc38861f12e58410997635/e2e/helpers.ts#L48-L55) in develop. You can use that here.